modules/i18n/src/main/java/com/devonfw/module/i18n/locale/api/LocaleResourceFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.devonfw.module.i18n.locale.api;
+
+import com.devonfw.module.i18n.locale.impl.LocaleNlsResourceImpl;
+import com.devonfw.module.i18n.locale.impl.LocaleResourceImpl;
+
+/**
+ * Factory for generating JSON according to implementation see {@link LocaleResourceImpl} ,
+ * {@link LocaleNlsResourceImpl}
+ *
+ */
+public interface LocaleResourceFactory {
+  /**
+   * @param locale
+   * @param filter
+   * @return
+   * @throws Exception
+   */
+  String getResourceAsJson(String locale, String filter) throws Exception;
```

Comment:
So what is the difference between `I18n` and `LocaleResourceFactory` except the name of the method?
I do not get this and it seems kind of odd to me.